package.json

```diff
@@ -435,7 +435,10 @@
       "ganache>keccak": false,
       "ganache>leveldown": false,
       "ganache>secp256k1": false,
-      "ganache>utf-8-validate": false
+      "ganache>utf-8-validate": false,
+      "gulp-watch>chokidar>fsevents": false,
```

Comment:
Curious how these started showing up now, but `fsevents` ensures that when we are watching files on macOS, we use the native file watcher built into the OS, so we may want to enable these as it might lead to less CPU load there.